### PR TITLE
Fixed default arguments for repacking llama3

### LINKED ISSUE #11327: Llama3.1 repacking requires additional arguments
**Describe the bug**

1. Repacking the scripts for llama3.1 requires additional arguments like num_layers and hidden_size. These should be default and not required to be passed specifically by the user.
2. params.json missing from the repacked directory.

**To Reproduce**
python models/demos/t3000/llama2_70b/scripts/repack_weights.py <path_to_checkpoint_dir> <repacked_output_dir> <chunk_size>

**Expected behavior**
The repacked_weights directory should be created without any errors.

**Screenshots**
image

**Environment information:**
- OS: Ubuntu 20.04

### PR DESCRIPTION
**Pull request by ksriv001**
### Ticket
GitHub Issue: #11327 
### Problem description
llama3.1 has a few additional changes that need to be added to repack the weights completely. 
Also, the arguments need to be set to default and not required by the user.
### What's changed
The changes involve copying a params.json from the actual checkpoint directory to the directory created for repacking weights.

### Checklist
- [ ] Post commit CI passes
- [ ] Blackhole Post commit (if applicable)
- [ ] Model regression CI testing passes (if applicable)
- [ ] New/Existing tests provide coverage for changes

### CODE REVIEW

File: models/demos/t3000/llama2_70b/scripts/repack_weights.py [hunk 1]
```diff
@@ -87,4 +92,4 @@ def repack(in_dir, out_dir, chunk_size, num_layers, hidden_size):
     parser.add_argument("num_layers", type=int, default=80, help="total number of layers")
     parser.add_argument("hidden_size", type=int, default=8192, help="hidden size of the model")
     args = parser.parse_args()
-    repack(args.in_dir, args.out_dir, args.chunk_size, args.num_layers, args.hidden_size)
+    repack(args.in_dir, args.out_dir, args.chunk_size)
```
Comment (cglagovichTT) [inline models/demos/t3000/llama2_70b/scripts/repack_weights.py:95]: Could you rebase and undo this change to the repack call? Someone merged a PR yesterday which fixed the default args issue. If you can pick that up and add your params.json copying, that would be great! https://github.com/tenstorrent/tt-metal/pull/11297#event-13859102211

**Comment by cglagovichTT** [comment]
Looks good to me

File: models/demos/t3000/llama2_70b/scripts/repack_weights.py (hunk 1 above)
Comment (ksriv001) [reply models/demos/t3000/llama2_70b/scripts/repack_weights.py]: Done, thanks!

**Review by cglagovichTT — APPROVED** [review]
